### PR TITLE
Valkyrize indexers

### PR DESCRIPTION
**Pull request by ShanaLMoore**
Update indexers with customizations

TODO: Specs? => they already exist. I'm having trouble running them locally. We can address when we fix the rest of the specs

### CODE REVIEW

File: app/indexers/cdl_resource_indexer.rb [hunk 1]
```diff
@@ -7,10 +7,10 @@ class CdlResourceIndexer < Hyrax::Indexers::PcdmObjectIndexer(CdlResource)
...
   # Uncomment this block if you want to add custom indexing behavior:
-  #  def to_solr
-  #    super.tap do |index_document|
-  #      index_document[:my_field_tesim]   = resource.my_field.map(&:to_s)
-  #      index_document[:other_field_ssim] = resource.other_field
-  #    end
-  #  end
+  def to_solr
+    super.tap do |index_document|
+      index_document['admin_note_tesim'] = resource.admin_note
```
Comment (kirkkwang) [inline app/indexers/cdl_resource_indexer.rb:12]: i actually don't think we need any of the `index_document['admin_note_tesim']` indexed explicitly, I _believe_ the .yaml should take care of it., might be good to check though 🤔

Comment (ShanaLMoore) [reply app/indexers/cdl_resource_indexer.rb:12]: oh man, i didn't even notice the yaml files. 👀

Comment (ShanaLMoore) [reply app/indexers/cdl_resource_indexer.rb:12]: it uses the metadata yml files like the models do? @kirkkwang

Comment (ShanaLMoore) [reply app/indexers/cdl_resource_indexer.rb:12]: In that case, i don't really think we need this PR 😭 Thanks for pointing that out, too bad I didn't notice that sooner.

File: app/indexers/generic_work_resource_indexer.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  include Hyrax::Indexer(:basic_metadata)
+  include Hyrax::Indexer(:bulkrax_metadata)
+  include Hyrax::Indexer(:generic_work_resource)
+  include Hyrax::Indexer(:with_pdf_viewer)
+  include Hyrax::Indexer(:with_video_embed)
+
+  include HykuIndexing
+
+  # Uncomment this block if you want to add custom indexing behavior:
+  def to_solr
```
Comment (ShanaLMoore) [inline app/indexers/generic_work_resource_indexer.rb:15]: Per your earlier comment, it seems like we need to add a generic_resource metadata yaml with admin not for generic work and image model. Would you agree @kirkkwang ?

**Pull request merged**